### ISSUE TITLE
Barcode Scanner

### ISSUE THREAD

**Issue by subhay389** [subhay389/ShoppingCart#1]
It is working perfectly fine when I tried to run the add in ionic view. But it does not run in android emulator. It showed the following error. 
<img width="1209" alt="screen shot 2015-11-27 at 2 25 04 pm" src="https://cloud.githubusercontent.com/assets/13982554/11454280/654fc21c-95f4-11e5-8142-91bd4ab99c06.png">

**Comment by ashoktamang** [comment]
I think bar code scanner should work only on actual devices not on emulator.

**Comment by subhay389** [comment]
Yes the barcode runs on devices only but the whole app should be able to run on the emulator which is not happening

**Comment by SalmonRanjay** [comment]
In you config.xml file you have an error delete the the error lines and try to run the project again. 
"=======> head
"
is one of them and there other merge conflict lines in your config.xml file
